src/python/examples/simple_grpc_infer_client.py

```diff
@@ -85,18 +85,68 @@
         required=False,
         default=None,
         help=
-        'The compression algorithm to be used when sending request to server. Default is None.'
+        'The compression algorithm to be used when sending request to server. '
+        'Default is None.'
+    )
+    # GRPC KeepAlive: https://github.com/grpc/grpc/blob/master/doc/keepalive.md
+    parser.add_argument(
```

Comment:
I think will be better to have another example to demonstrate keep alive settings. The simple_grpc_infer_client.py should be simple. These many new options pollute our simple example.

Let's capture this in `simple_grpc_keepalive_client.py` ?